packaging/hudi-presto-bundle/pom.xml

```diff
@@ -240,6 +241,13 @@
       <scope>${presto.bundle.bootstrap.scope}</scope>
     </dependency>
 
+    <!--httpclient needs to be updated to 4.5.13 due to vulnerability in lower versions -->
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <version>4.5.13</version>
```

Comment:
Similarly, see if `http.version` can be upgraded directly, instead of hardcoding the version here.

Reply:
@yihua placeholder _http.version_  is used for other modules and used in other dependencies version as well. Changing it globally will create issues.We are making the changes in packaging/hudi-presto-bundle only.